src/views/tutorial-view/components/variants-dropdown-disclosure/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { useRouter } from 'next/router'
+import DropdownDisclosure, {
+	DropdownDisclosureLinkItem,
+} from 'components/dropdown-disclosure'
+import Text from 'components/text'
+import {
+	getVariantParam,
+	getVariantPath,
+	TutorialVariantOption,
+} from 'views/tutorial-view/utils/variants'
+import { VariantsDropdownDisclosureProps } from './types'
+import s from './variants-dropdown-disclosure.module.css'
+
+// @TODO add aria-labelledBy
+
+export function VariantsDropdownDisclosure({
+	variant,
+}: VariantsDropdownDisclosureProps) {
+	const { asPath } = useRouter()
+	// @TODO hook this into useVariants hook once data is wired
+	const activeOption = variant.options[0]
+
+	return (
+		<div className={s.root}>
+			<Text weight="semibold" size={100} className={s.label}>
+				{variant.name}
+			</Text>
+			<DropdownDisclosure
+				aria-label={`${variant.name}: ${activeOption.name}`}
```

Comment:
Would love feedback on the right thing to do here. We have a visual label but currently with dropdown disclosure there isn't a way to associate the visual label with the list of dropdown disclosure items. I think that could be a good candidate for a refactor later on but feels outside the scope of this PR, so looking for the best case here, which could perhaps be using no aria label and just having the button text there, but then we lose the extra context of the parent variant name as the visual label.

Reply:
👍 I'm not sure, but seems like a possible case for adding support for an `aria-labelledby` prop to `DropdownDisclosure`? +1 on a separate PR for that, if we want. I think the `aria-label` here makes sense. Only thing I can think of is adding a comment here if we're relatively sure we wanna revisit it 🍞

Reply:
Yeah I agree, adding support for `aria-labelledby` would be the best end result here. I can make a follow up ticket and add a breadcrumb 🍞

Reply:
I agree with the approach here ^